UIKitDZ/UIKitDZ/ChangeTextViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 22.09.2022.
+//
+
+import UIKit
+/// Контроллер изменения текста
+class ChangeTextViewController: UIViewController {
+    // MARK: - Private Properties
+    let textLable = UILabel()
+    let sizeTextSlider = UISlider()
+    let textColorPicker = UIPickerView()
+    let amountLinesPicker = UIPickerView()
+    let amountLinesArray = [0, 1, 2]
+    let textColorArray = [UIColor.black, UIColor.orange, UIColor.red, UIColor.blue, UIColor.green]
```

Comment:
```suggestion
    let textColors = [UIColor.black, UIColor.orange, UIColor.red, UIColor.blue, UIColor.green]
```
убираем слово Array везде и забываем